### PR TITLE
Add info regarding issues with IPv6

### PR DESCRIPTION
**Pull request by Jacold**
Add info about IPv6 causing extreme lags when server don't support or isn't configured to handle IPv6 addresses.
https://github.com/GeyserMC/Geyser/issues/3207

### CODE REVIEW

**Comment by Camotoy** [comment]
I'm unaware of what APN is.

**Comment by Jens-Co** [comment]
I assume access point name. You should change APN to its full name, Otherwise it doesn't make much sense.

**Comment by Jacold** [comment]
^ done

File: _docs/geyser/common-issues.md [hunk 1]
```diff
@@ -39,6 +39,10 @@ This usually has something to do on your host's end. Most commonly, it's because
 
 **PLEASE NOTE:** If your server automatically redownloads jars upon startup, such as with an autoupdate system, this workaround will not work. Please contact your host if this does not work for you as there is nothing we can do.
 
+# Extreme lags on mobile data
+
+Check your Access Point Name (APN) - if you're using IPv6 and server don't support or isn't configured to handle IPv6 addresses you need to change APN to IPv4.
```
Comment (rtm516) [inline _docs/geyser/common-issues.md:44]:
```suggestion
Check your Access Point Name (APN) - if you're using IPv6 and the server doesn't support or isn't configured to handle IPv6 connections/addresses you need to change APN to IPv4.
```

**Review by rtm516 — CHANGES_REQUESTED** [review]

**Review by onebeastchris — APPROVED** [review]
While niche, could be important. lgtm

**Comment by Chew** [comment]
Hey there! Thank you for contributing to the wiki. We've migrated the wiki to our new [GeyserMC/GeyserWebsite](https://github.com/geysermc/geyserwebsite). If you'd like to, please migrate your PR to the new repo. From there, we can review it. Thank you for your contributions!

**Pull request closed without merging**